README.md

```diff
@@ -0,0 +1,51 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> This is a personal todo project that allows users to manage their daily tasks.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- WebPack
+
+
+## Getting Started
+For development, you are only required to have Node.js and a node global package, also Yarn, installed in your environment.
+
+## Node
+- ### Node Installation on Windows
+  
+  Go to [official Node.js website](https://nodejs.org/) and download the installer.
+Also, be sure to have `git` available in your PATH, `npm` might need it (You can find git [here](https://git-scm.com/)).
+
+- ### Node Installation on other Operating Systems
+  You can find more information about the installation on the [official Node.js website](https://nodejs.org) and the [official NPM website](https://npmjs.org/).
+
+
+### Install
+Go to the root folder and run the following:
+
+    $ git clone https://github.com/Paul-js-hub/todo-list
+    $ cd todo-list
+    $ npm install
+
+# Running the project
+
+   $ Open the index.html in your browser
+
+## Live link
+[Live Demo Link](Coming Soon)
+
+## Author
+👤 **Paul Onchera**
+
+- GitHub: [@githubhandle](https://github.com/Paul-js-hub/)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/paul-onchera-499a57144/)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] **Optional:** Your license leads to a 404. Kindly update it with [the template.](https://github.com/microverseinc/readme-template) Ensure to edit to match your purpose.